src/common/interfaces/IMessageSender.sol

```diff
@@ -1,9 +0,0 @@
-// SPDX-License-Identifier: GPL-2.0-or-later
-pragma solidity >=0.5.0;
-
-/// @notice Generic interface for entities that handles outgoing messages
-interface IMessageSender {
```

Comment:
Note, only implemented by the Gateway, and no mocks/tests use this, so 've moved to Gatewat because it fully depends on how Gateway works